client/packages/invoices/src/OutboundShipment/api/hooks/utils/useSelectedIds.ts

```diff
@@ -0,0 +1,20 @@
+import { useTableStore } from 'packages/common/src';
+import { useOutboundRows } from '../line/useOutboundRows';
+
+export const useSelectedIds = () => {
+  const { items, lines } = useOutboundRows();
+
+  const selectedIds =
+    useTableStore(state => {
+      const { isGrouped } = state;
+
+      return isGrouped
+        ? items
+            ?.filter(({ id }) => state.rowState[id]?.isSelected)
```

Comment:
do you think it's worth creating a refactor issue for later, to consolidate this sort of selected lines logic ? i.e. try searching for `?.filter(({ id }) => state.rowState[id]?.isSelected)`

Reply:
YES